### PR TITLE
Add responsive styling to each page, and clean up consolelogs.

### PR DESCRIPTION
**Pull request by corysanders3**
# Pull Request

## Description
- Add responsive design to each page
- Clean up code (remove unnecessary code / console.logs)

## Related Issues
- For the teas page, I did not alter how the grid was setup. I think due to all the different stylings, just changing the grid template columns and grid rows added some issues to the page layout. I did add some padding to it, so the card drops to the following row sooner. If we want, we can take a look at this as I think it would require styling updating each child element.

## Screenshots (if applicable)
Screenshot 2024-04-09 at 10 03 13 PM

## Testing
- Testing was not checked after updating css

## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [ ] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.

## Reviewer Instructions
- Let me know your thoughts on approaching the design of the tea cards section.

### CODE REVIEW

**Comment by AsherSpurr** [comment]
Love it! My only note is I think we need to lighten the background a bit, maybe lower(heighten?) the transparency.

**Review by AsherSpurr — APPROVED** [review]
Nicely done!

File: src/components/teas/Teas.css [hunk 1]
```diff
@@ -21,4 +21,8 @@
     margin-left: 20px
 }
 
-
+@media screen and (max-width: 400px) {
```
Comment (tednaphil) [inline src/components/teas/Teas.css:24]: NICE

File: src/components/home/Home.css [hunk 2]
```diff
@@ -48,4 +52,61 @@
...
+    width: 240px;
+}
+
+.creation-p {
+    margin: 20px 0px 5px 0px;
+    font-size: larger;
+    padding: 0px 20px 0px 20px;
+}
+
+@media screen and (max-width: 1116px) {
```
Comment (tednaphil) [inline src/components/home/Home.css:83]: I love how the home page looks on mobile view

**Review by tednaphil — APPROVED** [review]

**Pull request merged**